File: Backend/src/main/java/org/yatharth/infcube/model/ClientMessage.java
package org.yatharth.infcube.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class ClientMessage {
    public String type;
    public String roomId;
    public String playerId;
    public Object payload;
}

File: Assets/Scripts/Manager/Networking/Models/ClientMessage.cs
using System;

[Serializable]
public class ClientMessage
{
    public string type;
    public string roomId;
    public string playerId;
    public object payload;

    public ClientMessage() { }

    public ClientMessage(string type, string roomId, string playerId, object payload = null)
    {
        this.type = type;
        this.roomId = roomId;
        this.playerId = playerId;
        this.payload = payload;
    }
}
